mesonbuild/coredata.py

```diff
@@ -347,4 +347,6 @@ def get_builtin_option_default(optname):
                           'build.ninja': None,
                           'scan-build': None,
                           'reconfigure': None,
+                          'dist': None,
+                          'distcheck': None,
```

Comment:
Maybe. Originally I planned to have a distcheck target but then realized just doing it every time is simpler.